tests/units/common/test_sftp.py

```diff
@@ -23,15 +22,7 @@ def initiate_sftp_service():
     raises(DirectoryNotFoundException, initiate_sftp_service)
 
 
-def test_logs():
-    with capture_logs() as cap_logs:
+def test_error_raise():
```

Comment:
I don't understand why we need this?

Reply:
I tiried to explain it here: https://github.com/cern-sis/workflows/pull/92/files/76766090ae4f3fcc0a150d3a1644e49cc41acb99#r1100381724

I put the new return on condition, and wanted to test is it really throws an error.
If I woudn't return False in thi __exit__ function, it wouldn't throw an error, even if it was raised
docs: https://docs.python.org/3/reference/datamodel.html#object.__exit__

Reply:
Apologies, but still I don't understand why we need this test. It catches any error extending `Exception`, meaning almost everything

Reply:
I wanted to ensure that it catches whichever Exception and raises it. Before adding return False it was not raising anything, so the task was not stopped, even thought the connection so SFTP was not successful.